telco-hub/configuration/reference-crs/optional/lso/lsoSubscription.yaml

```diff
@@ -8,5 +8,5 @@ spec:
   channel: stable
   installPlanApproval: Automatic
   name: local-storage-operator
-  source: redhat-operators-disconnected
+  source: cs-redhat-operator-index-v4-17
```

Comment:
In other contexts we use a fixed catalogsource name (eg redhat-operators-disconnected) and update the index image field in that catalog for new versions (eg advancing from 4.17 to 4.18). With the version number in the name as you have here that would require us to update the catalog source in each subscription when upgrading (vs updating just the catalog). Is that intentional here or can the single catalog with changing index be used? (noting that we likely need to update the channel unless the subscriptions use "stable").

Reply:
I understand. I modified the `source` attribute in those subscriptions with the default catalog source name produced by the "oc mirror" command when mirroring the images. But I've seen that this can be overwritten with the `targetCatalog `attribute of the ImageSetConfiguration, so I will include this attribute and leave the source catalog as it was.